Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -110,6 +111,30 @@ class ApplicationManifestParserTest : public testing::Test {
         return parseString(manifestContent);
     }
 
+    ApplicationManifest parseShortcutFirstTopLevelProperty(const String& key, const String& value)
+    {
+        String manifestContent = "{ \"shortcuts\": [{\"" + key + "\": " + value + ", \"url\": \"example\" }]}";
```

Comment:
Please use makeString() and ASCIILiteral, like so:
`auto manifestContent = makeString("{ \"shortcuts\": [{\""_s, key, "\": "_s, value, ", \"url\": \"example\" }]}"_s);`

Reply:
Switched to using `makeString()` and ASCIILiteral for all newly added methods.